packages/proto-signing/src/msgs.ts

```diff
@@ -26,3 +26,19 @@ export class MsgSend extends Message {
   @cosmosField.repeatedMessage(3, Coin)
   public readonly amount?: readonly Coin[];
 }
+
+// TODO: move somewhere else as needed
+@registered(defaultRegistry, "/cosmos.auth.BaseAccount")
+export class BaseAccount extends Message {
```

Comment:
This type is now in the autogenerated code in @cosmjs/starport (where all the queries are) and not needed here anymore